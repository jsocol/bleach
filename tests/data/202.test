<div id="136"><form action="x" method="post">
<input name="username" value="admin" />
<input name="password" type="password" value="secret" />
<input name="injected" value="injected" dirname="password" />
<input type="submit">
</form>//["'`-->]]>]</div>
