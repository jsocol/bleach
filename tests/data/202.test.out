<div id="136"><form method="post" action="x">
<input value="admin" name="username">
<input value="secret" type="password" name="password">
<input value="injected" name="injected">
<input type="submit">
</form>//["'`--&gt;]]&gt;]</div>
